packages/core/src/components/Tooltip/index.tsx

```diff
@@ -17,6 +17,9 @@ const EMPTY_TARGET_RECT: ClientRect = {
   width: 0,
 };
 
+const MOUSE_ENTER_DELAY = 0;
+const MOUSE_LEAVE_DELAY = 0.1;
```

Comment:
is there a reason to make these seconds? typically I've seen these defined as `ms` because that's what you have to pass to timeout apis.